api/src/vfs/mod.rs

```diff
@@ -104,25 +149,31 @@ impl Node {
         )
     }
 
-    /// Gets the actual data of this node. This is only possible for files. If
-    /// called for a directory, will return
-    /// <ServerError::UnsupportedFileOperation>.
+    /// Gets the actual data of this node. This is only possible for files, and
+    /// requires read permission. Will return an error if this node is a
```

Comment:
I know im not a grammar expert buttt starting a sentence with will feels weird lol